share/zephyr-package/cmake/ZephyrConfig.cmake

```diff
@@ -11,17 +11,47 @@
 # It will be empty if not set in environment.
 
 macro(include_boilerplate location)
+  list(PREPEND CMAKE_MODULE_PATH ${ZEPHYR_BASE}/cmake/modules)
```

Comment:
Does this mean every user will have to `west zephyr-export` again after this PR is merged?

Are there any compatibility concerns when bisecting across the boundary of this PR's patches?

Reply:
> Does this mean every user will have to west zephyr-export again after this PR is merged?

No.
`west zephyr-export` is about CMake **package** registry, this is CMake **modules** and local to the actual package being invoked.

> Are there any compatibility concerns when bisecting across the boundary of this PR's patches?

No, none at all.
Not even for samples using old style `include(${ZERPHYR_BASE}/cmake/app/boilerplate.cmake)`.
Those will still work after this PR.

Reply:
Thanks for confirming. Resolved.